lego/apps/events/views.py

```diff
@@ -57,6 +59,12 @@
 class EventViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
 
     filter_class = EventsFilterSet
+    filter_backends = (
+        DjangoFilterBackend,
+        filters.OrderingFilter,
+        LegoPermissionFilter,
+    )
+    ordering_fields = ("start_time", "end_time", "title", "registration_deadline_hours")
```

Comment:
I dont think we need `registration_deadline_hours`. It is only a fixed integer count of hours before the events starts, that the registration closes.

Reply:
I thought maybe you might want to use this in combination with the start time for sorting after when the registration closes. But maybe its not so useful?